packages/astro/src/assets/build/remote.ts

```diff
@@ -19,6 +19,40 @@ export async function loadRemoteImage(src: string) {
 	return {
 		data: Buffer.from(await res.arrayBuffer()),
 		expires: Date.now() + expires,
+		etag: res.headers.get('Etag'),
+	};
+}
+
+export async function revalidateRemoteImage(src: string, etag: string) {
```

Comment:
It think it's valuable to document the business logic of the function.  For example, you used some headers, checked specific status codes and more. From a POV of a reviewer (me), I have no idea what's going on, so it's available to have this knowledge base written a JSDoc, other developers know that.

Reply:
I've added a JSDoc comment to this function that hopefully explains its purpose.